src/dreyfus_util.erl

```diff
@@ -166,6 +167,46 @@ time(Metric, {M, F, A}) when is_list(Metric) ->
         couch_stats:update_histogram([dreyfus | Metric],  Length)
     end.
 
+utc_timestamp() ->
```

Comment:
We didn't find any timestamp function in couch_util yet. I will modify it after it added in couch_util.